src/administration/integrations/gluescript.md

```diff
@@ -0,0 +1,251 @@
+For example: `project.environments.master` will give you access to the metadata of the master environment of the current project.
+
+The API starts at `api.project` and is based on a one resource == one object model.
+
+### Properties
+
+The properties of the resource can be access directly on the object, like:
+
+```javascript
+project.owner
+```
+
+### Sub-objects
+
+Sub-objects can be accessed directly on the object, like:
+
+```javascript
+project.settings
+```
+
+### Collections
+
+Resources that are collections support iteration:
+
+```javascript
+project.environments.forEach(function(env) {
+  console.log(env.name);
+});
+```
+
+They also support direct access to an element:
+
+```javascript
+project.environments["master"].name
+```
+
+### Actions
+
+Resource actions are accessible as methods on the object:
+
+```javascript
+project.environments["master"].branch({
+  title="Test",
+  name="test",
+})
+```
+
+## Activity API
+
+The activity API accessed through `activity` gives you all the metadata about the current activity that invoked the GlueScript. The schema is the same as the one for Webhooks, documented in the [Webhook section](https://docs.platform.sh/administration/integrations/webhooks.html) of our documentation.
+
+## Process API
+The process API gives you access to the environment in which the GlueScript is running, such as to environment variables.
+
+For example:
+
+`process.env.FOO` will give you access to the `FOO` environment variable. This is a great place to access API keys, or other secrets.
+
+## Storage API
+
+A storage API is offered for integrations to store information between calls. E.g. if you want to be able to edit a previous slack message. The storage is scoped per integration.
+
+### `storage.get(key)`: Get a key from the storage
+
+ * `key` (string): The key to get
+
+Returns `null` if the key doesn't exist.
+
+### `storage.set(key, value)`: Create or update a key into the storage
+
+ * `key` (string): The key to set
+ * `value` (string): The value to set, must be a string (will not be cast)
+
+### `storage.remove(key)`: Remove a key from the storage
+
+ * `key` (string): The key to remove
+
+No-op if the key doesn't exist
+
+### `storage.clear()`: Clear the storage
+
+## Fetch API
+
+### fetch(): launch an HTTP request
+
+The fetch API allows you to do HTTP calls to external services. This implements a limited, synchronous version of the Fetch API with a hardcoded timeout of 30s, for example:
+
+```javascript
+var resp = fetch(
+  "http://example.com/",
+  {
+    method: "POST",
+    body: JSON.stringify({
+        "test": "toto",
+    }),
+  }
+)
+
+if (!resp.ok) {
+    throw "unable to fetch";
+}
+```
+
+For more information see this [Mozilla developer network article on Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch).
+
+
+## Crypto API
+
+We provide a minimal set of Crypto APIs to enable you to do signed requests as many APIs will require you to sign things with an HMAC.
```

Comment:
Proposed revision:

> The GlueScript integration provides a minimal set of crypto APIs that you can use to sign requests. This is useful because many external APIs will require you to sign requests with an HMAC.